main_test.go

```diff
@@ -3,152 +3,209 @@ package main
 import (
 	"os"
 	"testing"
-
-	"github.com/atotto/clipboard"
+	"github.com/xyproto/randomstring"
 )
 
-func TestCopyToClipboard(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping clipboard test in CI/CD.")
+// TestParseContent_MultipleFiles verifies parseContent correctly concatenates contents of multiple files.
+func TestParseContent_MultipleFiles(t *testing.T) {
+	// Prepare test data
+	content1 := "Content from file 1"
+	content2 := "Content from file 2"
+	file1 := createTempFile(t, content1)
+	defer os.Remove(file1.Name())
+	file2 := createTempFile(t, content2)
+	defer os.Remove(file2.Name())
```

Comment:
As you moved to my recommendation, there is no need for this now as the tempdir where the file is created is deleted when the test ends

I spotted it here and in various place, look for remaining defer calls